02_introduction/exercise_2a.md

```diff
@@ -7,7 +7,7 @@
 
 ## Setup
 
-Create a file called `exercise_2a.go` in the `02_introduction/code` directory.
+Create a file called `exercise_2a.go` in the `code/02_introduction` directory.
```

Comment:
This was correct the way it was originally. The code folder doesn't exist which maybe have caused a bit of confusion.